modelci/cli/modelhub.py

```diff
@@ -196,3 +196,39 @@ def detail(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.get(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         data = r.json()
         model_detailed_view(MLModel.parse_obj(data))
+
+
+@app.command('convert')
+def convert(
+        way: str = typer.Argument(..., help= 'the type of converting, for example \'keras2onnx\''
```

Comment:
it is rather to use `from` which format and `to` which format

Reply:
also reconsider the argument name, they are quite confusing: `save`? `saved_path`?